src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const entryComponents = props.entries.map((entry) => {
+        return (
+            <ChatEntry
+                key={`${entry.sender}-${entry.timeStamp}`}
```

Comment:
Excellent key design!

Reply:
I guess if someone was able to send two messages in the exact same second, this would cause issues, but in a more realistic scenario these timestamps would have at least milliseconds, so it's much less likely that two messages have the exact same timestamp.